packages/node_modules/pouchdb-adapter-indexeddb/src/util.js

```diff
@@ -9,7 +9,7 @@ import {
 import { sanitise } from './rewrite';
 
 var DOC_STORE = 'docs';
-var META_STORE = 'meta';
+var META_LOCAL_STORE = 'meta';
```

Comment:
update specificity
```suggestion
const META_LOCAL_STORE = 'meta';
```

Reply:
I've deliberately kept the diff for this PR as small as possible to (1) make review simpler, and (2) try to prevent incidental performance changes.  I look forward to modernising the JS once this PR is merged (or rejected :grin:).